### PR TITLE
ensure current_link is not equal to new_link

### PR DESCRIPTION
**Pull request by shiboying**

### CODE REVIEW

**Comment by shiboying** [comment]
@bboe This is the pull request for changing the assertion of `current_link` and `new_link`.

File: test/controllers/pictures_controller_test.rb [hunk 1]
```diff
@@ -120,10 +120,10 @@ class PicturesControllerTest < ActionController::TestCase
   test 'can update picture url' do
     picture = create_pictures.first
     current_link = picture.link
-    new_link = 'http://thisis.anewlink.com'
+    new_link = "http://thisis.anewlink.com/#{current_link}"
```
Comment (bboe) [inline test/controllers/pictures_controller_test.rb]: Will this have two `http://` portions inside of it?

File: test/controllers/pictures_controller_test.rb [hunk 2]
```diff
@@ -140,10 +140,10 @@ class PicturesControllerTest < ActionController::TestCase
   test 'cannot update picture if input invalid' do
     picture = create_pictures.first
     current_link = picture.link
-    new_link = 'abc'
+    new_link = "abc/#{current_link}"
```
Comment (bboe) [inline test/controllers/pictures_controller_test.rb]: In this case I don't think `new_link` needs to derive from current_link (that will work though). Rather, we can assert that the update didn't happen (replace 148 with `assert_not_equal`).

File: test/controllers/pictures_controller_test.rb (hunk 1 above)
Comment (shiboying) [inline test/controllers/pictures_controller_test.rb]: Yes, I see. A ```new_link = "#{current_link}/abc" seems better to form a real new link.

File: test/controllers/pictures_controller_test.rb (hunk 2 above)
Comment (shiboying) [inline test/controllers/pictures_controller_test.rb]: like replay it with`assert_not_equal new_link, picture.reload.link`?

Comment (bboe) [inline test/controllers/pictures_controller_test.rb]: Yes.

**Comment by shiboying** [comment]
@bboe I have a new commit for that.

**Pull request merged**